swift/ios-shoppe-demo/Views/StoreViewController.swift

```diff
@@ -50,7 +50,7 @@ class StoreViewController: UICollectionViewController, UICollectionViewDelegateF
         navigationItem.title = "iOS Shoppe"
         navigationItem.rightBarButtonItem = barCartButton
 
-        collectionView.backgroundColor = .white
+        collectionView.backgroundColor = UIColor().fsBackground()
```

Comment:
Just curious, which color is this?  Is it one from the swatch?

Reply:
I'm not sure what the swatch is but its a dark navy color I got from our power point intro slides.

Reply:
Probably the same, but you might check https://fullstory.slack.com/files/UNTS3T26Q/F011MCK62F9/image.png and then convert the hex to RGB.  It's probably `.enterprise` but naming it in a code comment would help identify which color the RGB value applies to.

Reply:
I will post a separate pr for this :)